server/views/404.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Page Not Found :(</title>
+    <style>
```

Comment:
Can you create a new css file (for example : 404.css).
I think it is better to separate the style from a CSS file.